contrib/debian/install_go.sh

```diff
@@ -49,7 +49,7 @@ install_go() {
 
 	if [ -z "${NETDATA_DISABLE_GO+x}" ]; then
 		echo >&2 "Install go.d.plugin"
-		ARCH=$(uname -m)
+		ARCH="${DEB_TARGET_ARCH}"
```

Comment:
We need to update things to drop the ARCH_MAP stuff. Debian uses the same architecture names as Go in _most_ cases (`i386` is the only exception that matters currently, and we just need to strip the `i`).